app/assets/javascripts/vote_ajax.js

```diff
@@ -0,0 +1,21 @@
+$(document).ready(function() {
+  $("a.vote_change").click(function(event) {
+    event.preventDefault();
+
+    var request = $.ajax({
+			method: "GET",
+			url: $(this).attr('ajax_path')
+		});
+
+    request.done(function(data) {
+      if ('new_score' in data) {
+        $('li#review-rating-' + data.review_id).text("Review rating: " + data.new_score);
+      } else {
+        if ($('div.flash_message').text != data.login_error) {
+           $('div.flash_message').text(data.login_error)
```

Comment:
Fixed the quotes issue.  Keeping the variables named in Ruby style since that's where they originate - I'll just get different hound errors for changing them.